backend/src/bundles/user-details/user-details.controller.ts

```diff
@@ -128,6 +129,18 @@ class UserDetailsController extends ControllerBase {
                     }>,
                 ),
         });
+
+        this.addRoute({
+            path: UserDetailsApiPath.$ID,
+            method: 'GET',
+            handler: (options) => {
+                return this.findByUserId(
+                    options as ApiHandlerOptions<{
+                        params: { userId: string };
```

Comment:
I think we need Dto type for this
```suggestion
                        params: Dto;
```